### PR TITLE
Error fetching the Login URL

### PR DESCRIPTION
**Pull request by jackhuynh95**
https://github.com/arjusmoon860/strapi-google-auth/issues/12#issuecomment-1427081431

image

### CODE REVIEW

File: server/services/google.js
Comment (vicgmail) [inline server/services/google.js:1]: Owner, please add these changes

**Review by magnv — APPROVED** [review]
these changes would fix error SyntaxError: Unexpected token 'h', "https://ww/"... is not valid JSON

File: server/services/google.js
Comment (jackhuynh95) [reply server/services/google.js:1]: Owner approved and please merge this PR @vicgmail

**Comment by jackhuynh95** [comment]
> these changes would fix error SyntaxError: Unexpected token 'h', "https://ww/"... is not valid JSON

Can you also merge this PR @vicgmail

**Comment by arjusmoon860** [comment]
This isn't valid, as the plugin is supposed to work only with valid credentials & scopes. If none of these are present, there is already a condition check to reject the missing scope.

**Pull request closed without merging**